frontend/android/fastlane/Fastfile

```diff
@@ -82,18 +82,19 @@ platform :android do
 
     env_prod = sh("echo -n environment=production | base64")
 
-    keystore_options = {}
+    keystore_key_alias = ""
+    keystore_key_password = ""
 
     case build_config_name
     when 'nuernberg'
-      keystore_options[:KEYSTORE_KEY_ALIAS] = KEYSTORE_KEY_ALIAS_NUERNBERG
-      keystore_options[:KEYSTORE_KEY_PASSWORD] = KEYSTORE_KEY_PASSWORD_NUERNBERG
+      keystore_key_alias[:KEYSTORE_KEY_ALIAS] = KEYSTORE_KEY_ALIAS_NUERNBERG
+      keystore_key_password[:KEYSTORE_KEY_PASSWORD] = KEYSTORE_KEY_PASSWORD_NUERNBERG
```

Comment:
:x: Sorry, maybe my suggestion was wrong, but this will properly break now:
```suggestion
      keystore_key_alias = KEYSTORE_KEY_ALIAS_NUERNBERG
      keystore_key_password = KEYSTORE_KEY_PASSWORD_NUERNBERG
```

Reply:
yeah, you are right,i messed up here. fixed it.